src/components/NcAppNavigation/NcAppNavigation.vue

```diff
@@ -59,15 +59,17 @@ emit('toggle-navigation', {
 		role="navigation"
 		:class="{'app-navigation--close':!open }">
 		<NcAppNavigationToggle :open="open" @update:open="toggleNavigation" />
-		<slot />
 
-		<!-- List for Navigation li-items -->
-		<ul class="app-navigation__list">
-			<slot name="list" />
-		</ul>
+		<div :aria-hidden="ariaHidden" class="app-navigation__content">
+			<slot />
+			<!-- List for Navigation li-items -->
+			<ul ref="navigationList" class="app-navigation__list">
+				<slot name="list" />
+			</ul>
 
-		<!-- Footer for e.g. AppNavigationSettings -->
-		<slot name="footer" />
+			<!-- Footer for e.g. AppNavigationSettings -->
+			<slot ref="footer" name="footer" />
```

Comment:
Same here
```suggestion
			<slot name="footer" />
```

Reply:
Also, it is not possible to set ref on a slot. `<slot>` is a special template tag, not an HTML element or a Vue component.